### ISSUE TITLE
make_promise(...) is not a function

### ISSUE THREAD

**Issue by ekkis** [RangerMauve/cypher-promise#2]
I don't know where this is coming from:

> Unhandled rejection TypeError: make_promise(...) is not a function
>     at /.../node_modules/cypher-promise/index.js:59:48
>     at tryCatcher (/.../node_modules/bluebird/js/main/util.js:26:23)
>     at Promise._settlePromiseFromHandler (/.../node_modules/bluebird/js/main/promise.js:507:31)
>     at Promise._settlePromiseAt (/.../node_modules/bluebird/js/main/promise.js:581:18)
>     at Promise._settlePromises (/.../node_modules/bluebird/js/main/promise.js:697:14)
>     at Async._drainQueue (/.../node_modules/bluebird/js/main/async.js:123:16)
>     at Async._drainQueues (/.../node_modules/bluebird/js/main/async.js:133:10)
>     at Immediate.Async.drainQueues [as _onImmediate](/.../node_modules/bluebird/js/main/async.js:15:14)
>     at processImmediate [as _immediateCallback](timers.js:383:17)

especially since there is no line 59 in `index.js`.  I've grepped through both cypher-promise and bluebird for `make_promise` but I don't find a reference.  help?

**Comment by ekkis** [comment]
I was looking at the `index.js` for a different module.  the problem seems to be that `make_promise` doesn't return a function.  it returns a Promise object.  yet an attempt to call it is made.  the lines below are a little modified from the source to show what is being returned:

```
68                 var p = make_promise(multi.exec.bind(multi));
69 console.log(p); 
70                 return p();
```

and that returns:

```
Promise {
  _bitField: 0,
  _fulfillmentHandler0: undefined,
  _rejectionHandler0: undefined,
  _progressHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined,
  _settledValue: undefined }
```

which is clearly not a function and can't be called.  how is this supposed to work? no, I didn't register a promise implementation so it's using the default

**Comment by RangerMauve** [comment]
Thanks for the details! I'm currently on vacation, but I'll get to this this weekend

**Comment by RangerMauve** [comment]
Looks like this is a problem [in this line](https://github.com/RangerMauve/cypher-promise/blob/master/index.js#L59) . I'm gonna touch it up and add some unit tests.

**Comment by RangerMauve** [comment]
Fixed in #1.0.2
